apps/judicial-system/api/src/app/modules/user/user.model.ts

```diff
@@ -1,79 +1,24 @@
-import {
-  Column,
-  CreatedAt,
-  DataType,
-  Model,
-  Table,
-  UpdatedAt,
-} from 'sequelize-typescript'
+import { Field, ObjectType } from '@nestjs/graphql'
 
-import { ApiProperty } from '@nestjs/swagger'
+import { User as TUser, UserRole } from '@island.is/judicial-system/types'
 
-import { UserRole } from './user.types'
+@ObjectType()
+export class User implements TUser {
+  @Field()
```

Comment:
ditto (won't comment more on this, you can check the rest of your types)

Reply:
Við erum ekki að skila user id úr grunninum heldur bara kennitölunni. Finnst þér ástæða til að merkja kennitöluna með ID?

Reply:
Yup, the Apollo store needs to be able to match future objects to the objects in its cache, using the GraphQL ID provided.

Reply:
Will modify then, probably return the surrogate key instead.

Reply:
Modified and pushed.